### PR TITLE
Section 5 of Community page

### LINKED ISSUE #1271: New page design implementation for Contributors page
#### Current State of the Meshery.io site

Currently the page https://meshery.io/community/ is empty

#### Desired State of the Meshery.io site

Need to implement the mockup for this page from [Figma design](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=16435%3A39494&mode=design&t=6qls5jOGn5gJKKgm-1)
Contributors can pick separate section to implement the page fast

- [ ] Section-1: Banner section

image

- [ ] Section- 2: Resources section

image

- [ ] Section-3: Handbook section

image

- [ ] Section-4: Discuss forum section

image

- [ ] Section-5: Meshmate section

image

- [ ] Section-6: Calendar section"

image

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by lanzkrish**
**Description**
Fixes #1271

Added Section 5 of the Community page

**Notes for Reviewers**

link to the preview:
https://master--serene-medovik-1fae20.netlify.app/community/
image

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhanshutech** [comment]
@lanzkrish please link the pr with respective issue

**Comment by lanzkrish** [comment]
I can't link this pr as I don't have write access. I think I have to close this pull request and create a new one

**Comment by lanzkrish** [comment]
Screenshot 2023-07-12 130317
Screenshot 2023-07-12 130305
I don't have acces to link the pr with the issue #1271. Need assistance with this.

**Comment by Mohith234** [comment]
@lanzkrish Just edit the first comment and mention the issue

`Fixes #1271`

**Comment by Gautam-666** [comment]
@lanzkrish If you can add screenshots that would be great because the preview is not working.

**Comment by lanzkrish** [comment]
Thank you for your help @Mohith234

File: Gemfile.lock [hunk 1]
```diff
@@ -72,6 +72,8 @@ GEM
       rake (>= 10.0.0)
     sass-embedded (1.57.1-arm64-darwin)
       google-protobuf (~> 3.21)
+    sass-embedded (1.57.1-x64-mingw-ucrt)
+      google-protobuf (~> 3.21)
```
Comment (sudhanshutech) [inline Gemfile.lock:76]: why there is a change in gemfile

File: community.html [hunk 2]
```diff
@@ -4,7 +4,37 @@
...
+              data-logo-for-dark="/assets/images/logos/meshmate-stack-colorMode_svg__Layer_1.svg"
+              data-logo-for-light="/assets/images/logos/meshmate-stack-colormode_svg__layer_1_dark.png"
+              alt="Meshmate Logo"
+              id="logo-dark-light"
+            />
+          </a>
+        </div>
+      </div>
+  </div>
+  <a href="#" class="text highlight" style="padding: 10px;  margin-left: 2%;">
```
Comment (sudhanshutech) [inline community.html:37]:
```suggestion
  <a href="https://layer5.io/community/meshmates" class="text highlight" style="padding: 10px;  margin-left: 2%;">

```

Comment (sudhanshutech) [reply community.html:37]: it should redirect to this link

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (lanzkrish) [reply Gemfile.lock:76]: I think it happened during Jekyll's updation on my local machine. I mistakenly committed that without reverting.

**Review by lanzkrish — COMMENTED** [review]
gemfile has been reverted to original
link to the meshmates button has been added

**Pull request closed without merging**